repo.go

```diff
@@ -415,10 +415,18 @@ func (r *Repo) AddTargetsWithExpires(paths []string, custom map[string]interface
 }
 
 func (r *Repo) RemoveTarget(path string) error {
-	return r.RemoveTargetWithExpires(path, data.DefaultExpires("targets"))
+	return r.RemoveTargets([]string{path})
+}
+
+func (r *Repo) RemoveTargets(paths []string) error {
+	return r.RemoveTargetsWithExpires(paths, data.DefaultExpires("targets"))
 }
 
 func (r *Repo) RemoveTargetWithExpires(path string, expires time.Time) error {
+	return r.RemoveTargetsWithExpires([]string{path}, expires)
+}
+
+func (r *Repo) RemoveTargetsWithExpires(paths []string, expires time.Time) error {
```

Comment:
Ok I added a comment, but I'm not sure how a `SetTargets` method relates? Are you proposing something like `tuf set <paths>...`?